test/integration_test.go

```diff
@@ -4980,9 +4980,14 @@ func (ts *IntegrationTestSuite) TestScheduleUpdate() {
 		err = handle.Delete(ctx)
 		ts.NoError(err)
 	}()
+
+	stringKey := temporal.NewSearchAttributeKeyString("CustomStringField")
+	sa := temporal.NewSearchAttributes(stringKey.ValueSet("CustomStringFieldValue"))
+
 	updateFunc := func(input client.ScheduleUpdateInput) (*client.ScheduleUpdate, error) {
```

Comment:
Thinking about use cases ,I think the `ScheduleDescription` should get a `TypedSearchAttributes` field (in addition to the raw one already present) with the high-level typed search attributes container/collection, and that field could just be passed here as a pointer or slightly mutated then passed here as a pointer. Think about the use case where a user just wants to alter one search attribute but leave all the rest alone. How would that look?

Reply:
> Think about the use case where a user just wants to alter one search attribute but leave all the rest alone. How would that look?

This is a replacement, so they would need to pass in all SAs they want applied to the schedule, not just new ones.

Reply:
Added TypedSearchAttributes